src/compiler/propagation/type_primitive_wifi.cc

```diff
@@ -28,10 +28,10 @@ TYPE_PRIMITIVE_ANY(setup_ip)
 TYPE_PRIMITIVE_ANY(disconnect)
 TYPE_PRIMITIVE_ANY(disconnect_reason)
 TYPE_PRIMITIVE_ANY(get_ip)
-TYPE_PRIMITIVE_ANY(get_rssi)
 TYPE_PRIMITIVE_ANY(init_scan)
 TYPE_PRIMITIVE_ANY(start_scan)
 TYPE_PRIMITIVE_ANY(read_scan)
+TYPE_PRIMITIVE_ANY(get_ap_info)
```

Comment:
We might as well start teaching the type propagator that these primitives have very well defined return types:

```suggestion
TYPE_PRIMITIVE_ARRAY(ap_info)
```